02-arrays/exercise/08SortArrayBy2Criteria.js

```diff
@@ -0,0 +1,3 @@
+function sortArrayBy2Criteria(data) {
+    console.log(data.sort((a, b) => a.length !== b.length ? a.length - b.length : a.localeCompare(b)).join("\n"))
```

Comment:
IMO making the solution in one line doesn't make it more functional or good. I would suggest you to to try making it understandable and readable.

A few things that can help you to do so:

- Move the sorting algorithm into separate function
- Add variables for the comparisons and simplify the ternary operator